packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -144,12 +157,14 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
    */
   async _applyAnchor(
     context: Context,
-    commit: AnchorCommit,
+    commit: any,
```

Comment:
woah, moving in the wrong direction here.  More type data, not less please

Reply:
Yes, agreed, but I'd have to go to less type data regardless: `CeramicData | CommitData`.

To do `AnchorCommit | CommitData` I would have to check whether this was an anchor commit in `applyCommit` then `as AnchorCommit` it but pass it through unchanged if it was `CommitData`, which made me cringe to think of implementing.